src/components/Viewcomments.jsx

```diff
@@ -0,0 +1,42 @@
+import { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+
+function Viewcomments() {
+const { article_id } = useParams();
+console.log(article_id)
+const [currComments, setCurrComments] = useState(''); 
+const [isLoading, setIsLoading] = useState(true);
+useEffect(() => {
+    setIsLoading(true);
+    fetch(`https://news-be-heroku.herokuapp.com/api/articles/${article_id}/comments`)
+      .then((res) => {
+        console.log(res, 'RESjjjjjjjjjjjjj')
```

Comment:
Watch out for left over console logs, this will need to be removed in a refactor